python/Ganga/Core/GangaRepository/VStreamer.py

```diff
@@ -18,9 +18,12 @@
 from .GangaRepository import SchemaVersionError
 
 import xml.sax.saxutils
+import copy
 
 logger = getLogger()
 
+_eval_strings = {}
```

Comment:
I don't see a problem with globals here (that doesn't suggest a larger problem elsewhere) a cache name is something likely to come in a revision :)